pkg/envinfo/LocalConfigProviderMocks.go

```diff
@@ -0,0 +1,117 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: ./pkg/envinfo/envinfo.go
+
+// Package envinfo is a generated GoMock package.
+package envinfo
```

Comment:
Mocks generated for 
https://github.com/openshift/odo/blob/a28e37838c5cbaa396f6675fe90a9986168165f5/pkg/envinfo/envinfo.go#L90-L97 
 using gomock